socs/agents/meinberg_syncbox/agent.py

```diff
@@ -61,6 +65,25 @@ def _extract_oid_field_and_value(get_result):
     if not isinstance(oid_value, (int, bytes, str)):
         oid_value = None
 
+    if (field_name.split('_')[0] in CONVERT_OIDS) and (oid_value is not None):
+        unit_multipliers = {
+            "s": 1000000,   # seconds to microseconds
+            "sec": 1000000,  # seconds to microseconds
+            "ms": 1000,     # milliseconds to microseconds
+            "us": 1,        # microseconds
+            "ns": 0.001     # nanoseconds to microseconds
+        }
+
+        value, unit = oid_value.split()
+        value = float(value)
+
+        # Normalize to microseconds
+        if unit in unit_multipliers:
+            oid_value = value * unit_multipliers[unit]
+
+        # Change the field name to differentiate from raw output
+        field_name = field_name + "_value"
```

Comment:
This is good, and necessary for the InfluxDB type collision issue. However it might be better to add three new fields that are published/kept in `session.data`, rather than outright replace them.

I'm thinking mostly for compatibility with any existing code moving forward. For instance, we now have some code that parses the existing strings for the plots that @msilvafe made. As we continue to compare these old instances of timing issues to new ones that same code will likely be used still, given the old instances will always need it.

Otherwise this should be listed as a breaking change (I checked that box on your PR just now), since these fields will just disappear in the data files once this is deployed.